src/Services/BitapsTransaction.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+
+namespace App\Services;
+
+use PostMix\LaravelBitaps\Models\Transaction;
+use \Carbon\Carbon;
+
+
+class BitapsTransaction
+{
+    public function newTransaction($data)
+    {
+        $date = Carbon::now(); //$date->getTimestamp()
+        $status = $data['confirmations'] < 3 ? 'pending' : 'confirmed';
```

Comment:
@ert-warol, don't use a hard-coded number.
I don't think `BitapsTransaction` should be created at all, but anyway, the confirmation number is a part of business logic. So, it should not be placed on that repository. You can just allow this to be passed as a param.